pkg/controller/admissionchecks/multikueue/admissioncheck.go

```diff
@@ -267,7 +267,7 @@ func (m *mkClusterHandler) Create(ctx context.Context, event event.CreateEvent,
 		return
 	}
 
-	if err := queueReconcileForConfigUsers(ctx, mkc.Name, m.client, q); err != nil {
+	if err := m.queue(ctx, mkc, q); err != nil {
```

Comment:
Doesn't the fact that we could not detect this bug in the test indicate that we don't have any cases to verify here?

Reply:
The problem is that we don't test (and I don't think it's easy to do) the create only scenario, after the cluster is created one of the first things happening is for it's active condition to be set, triggering an Update which worked well.

Reply:
Uhm, couldn't add a unit test for creation?

Reply:
That will be the first if it's king in the entire project.

Reply:
> That will be the first if it's king in the entire project.

@trasc Alright. Could you create tracking issue?

/lgtm
/approve